livingdocs-server-base/18.Dockerfile

```diff
@@ -4,10 +4,4 @@
 #   RUN npm ci
 #   ADD . /app
 FROM livingdocs/node:18
-RUN apk add --no-cache imagemagick
```

Comment:
please no breaking change in minor updates

Reply:
Looks like it took the latest `libwebp` with imagemagick, I pushed the new images.